cranelift/egraph/src/ctxhash.rs

```diff
@@ -134,6 +134,7 @@ impl<K, V> CtxHashMap<K, V> {
 
     /// Return an Entry cursor on a given bucket for a key, allowing
     /// for fetching the current value or inserting a new one.
+    #[inline(always)]
```

Comment:
Can't just be `#[inline]`? Needs to be inlined in non-optimized builds too?

Reply:
I'd prefer not to affect non-optimized builds, but for whatever reason the `inline(always)` seemed to be necessary to get inlining to happen in *optimized* builds, unfortunately...

(I'd really love to gain more insight into the heuristics at play here!)